demisto_sdk/tests/test_files/non-valid-integration-test-not-configured.yml

```diff
@@ -0,0 +1,609 @@
+description: Alert and notify users using PagerDuty
+configuration:
+- display: API Key
+  name: APIKey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Service Key (for triggering events only)
+  name: ServiceKey
+  defaultvalue: ""
+  type: 0
+  required: false
+script:
+  feed: True
+  script: |
+    ''' IMPORTS '''
+    import os
+    import json
+    import requests
+    import datetime
+
+    ''' GLOBAL VARS '''
+    # Pagerduty API works only with secured communication.
+    USE_SSL = True
+
+    API_KEY = demisto.params()['APIKey']
+    SERVICE_KEY = demisto.params()['ServiceKey']
+
+    SERVER_URL = "https://api.pagerduty.com/"
+
+    DEFAULT_HEADERS = {
+                          'Authorization': 'Token token=' + API_KEY,
+                          'Accept': 'application/vnd.pagerduty+json;version=2'
```

Comment:
Do we really need all the contents of this big file if really all you're using it for is checking the `tests` field?